core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -302,12 +403,12 @@ import scala.compat.java8.FutureConverters._
             log.debug("[{}] CurrentQuery: No query polling", stageUuid)
         }
 
-        settings.eventsByTagCleanUpPersistenceIds match {
-          case duration: FiniteDuration =>
-            log.debug("Dropping metadata for persistence ids every {}", duration.pretty)
-            schedulePeriodically(PersistenceIdsCleanup, duration)
-          case _ =>
+        def optionallySchedule(key: Any, duration: Option[FiniteDuration]): Unit = {
+          duration.foreach(fd => schedulePeriodically(key, fd))
         }
+
+        optionallySchedule(PersistenceIdsCleanup, settings.eventsByTagCleanUpPersistenceIds)
+        optionallySchedule(ScanForDelayedEvents, settings.eventsByTagBacktrack.interval)
```

Comment:
Where's the slow interval?

Reply:
THere's only one schedule, and when it triggers it decides to do a short or long backtrack. I didn't want multiple back tracks going on at the same time